docker/manage

```diff
@@ -216,7 +216,7 @@ configureEnvironment() {
 
   # identity-controller
   export IDENTITY_CONTROLLER_PORT=5000
-  export IDENTITY_CONTROLLER_HOOK_PORT=5050
+  export IDENTITY_CONTROLLER_HOOK_PORT=5000
```

Comment:
Do we want to have the webhook port be the same as the API port? If we do, we shoud remove one of the variables since it becomes obsolete.